src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/WalletForm.tsx

```diff
@@ -69,8 +69,7 @@ export const WalletForm = ({ signatories, onContinue, isActive, isLoading, onGoB
     onSubmit(name, threshold.value, creator.accountId);
   };
 
-  const hasNoAccounts =
-    wallets.filter((wallet) => !walletUtils.isWatchOnly(wallet) || !walletUtils.isMultisig(wallet)).length === 0;
+  const hasNoAccount = !wallets.some((wallet) => !walletUtils.isWatchOnly(wallet) || !walletUtils.isMultisig(wallet));
```

Comment:
I took the liberty to change a check for `filter().length === 0` by a `some()` because it's more efficient as it will stop iterating as soon as it's false.

Reply:
even better, thank you